### DISCUSSION TITLE
Newsletter sending already in progress, need to finish first.

### DISCUSSION THREAD

**Discussion by mediafant** [pimcore/pimcore#15885] [Q&A / Support] [answered]
Hi all,

in Pimcore 10.6.9 I've create a newsletter. Testsending works perfect. But if I "send it NOW" there is this message:
```
Status: 500 | Internal Server Error
URL: /admin/newsletter/send
Method: post
Message: Newsletter sending already in progress, need to finish first.
```

The Cronjob to `/bin/console pimcore:maintenance -a` works.

How can I fix this problem?

Thanks for hints 
Andreas

**Comment by kingjia90** [comment]
Can you try with `bin/console messenger:consume pimcore_core`? 
see https://github.com/pimcore/newsletter-bundle/blob/b01f6894c60f1690a36228454c501c3545ef53f0/config/pimcore/default.yaml#L21

to process the queue
https://pimcore.com/docs/platform/Pimcore/Getting_Started/Installation/Webserver_Installation#5-maintenance-cron-job

**Comment by mediafant** [comment, marked as answer]
thanks for hint, but output of:
`./bin/console messenger:consume pimcore_core -vv`

is
```
14:11:41 INFO      [messenger] Pimcore\Messenger\AssetUpdateTasksMessage was handled successfully (acknowledging to transport). ["class" => "Pimcore\Messenger\AssetUpdateTasksMessage"]
14:11:44 INFO      [messenger] Received message Pimcore\Messenger\SearchBackendMessage ["class" => "Pimcore\Messenger\SearchBackendMessage"]
14:11:44 INFO      [messenger] Message Pimcore\Messenger\SearchBackendMessage handled by Pimcore\Messenger\Handler\SearchBackendHandler::__invoke ["class" => "Pimcore\Messenger\SearchBackendMessage","handler" => "Pimcore\Messenger\Handler\SearchBackendHandler::__invoke"]
```
(all seconds)

but there is no sending of mails and if I want send a new newsletter there is always the same error-message from before
`Newsletter sending already in progress, need to finish first.`

As I had written, I'm using Pimcore 10.6.9, for this there is no Newsletter-Bundle (only for Pimcore 11), so I think it must be a part of Pimcore, is it?

thanks for any idea?

Andreas

**Reply by mediafant** [reply]
For reset, respectively for delete old jobs, it is possible to delete newsletter-item in Database "tmp_store".

But I have not found a way to send newsletter.
